django_celery_results/migrations/0012_alter_taskresult_status.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.5 on 2023-09-04 14:56
```

Comment:
I think we need to remove this and regenerate again after pulling from main branch

Reply:
Done; I've rebased and created new migrations.